# HG changeset patch
# User David Shea <dshea@redhat.com>
# Date 1398359901 14400
#      Thu Apr 24 13:18:21 2014 -0400
# Node ID c1db9bf8fcb9b110986ffbcd848792d48a251ffb
# Parent  e4328f29810f69979025297dc54bf69f5ad53429
Change --init-hooks back to --init-hook

diff -r e4328f29810f -r c1db9bf8fcb9 lint.py
--- a/lint.py	Tue Apr 22 20:49:52 2014 +0300
+++ b/lint.py	Thu Apr 24 13:18:21 2014 -0400
@@ -907,7 +907,7 @@
         try:
             preprocess_options(args, {
                     # option: (callback, takearg)
-                    'init-hooks':   (cb_init_hook, True),
+                    'init-hook':    (cb_init_hook, True),
                     'rcfile':       (self.cb_set_rcfile, True),
                     'load-plugins': (self.cb_add_plugins, True),
                     })
diff -r e4328f29810f -r c1db9bf8fcb9 test/unittest_lint.py
--- a/test/unittest_lint.py	Tue Apr 22 20:49:52 2014 +0300
+++ b/test/unittest_lint.py	Thu Apr 24 13:18:21 2014 -0400
@@ -381,9 +381,9 @@
 
     def test_init_hooks_called_before_load_plugins(self):
          self.assertRaises(RuntimeError,
-                           Run, ['--load-plugins', 'unexistant', '--init-hooks', 'raise RuntimeError'])
+                           Run, ['--load-plugins', 'unexistant', '--init-hook', 'raise RuntimeError'])
          self.assertRaises(RuntimeError,
-                           Run, ['--init-hooks', 'raise RuntimeError', '--load-plugins', 'unexistant'])
+                           Run, ['--init-hook', 'raise RuntimeError', '--load-plugins', 'unexistant'])
 
 class ConfigTC(TestCase):
 
